examples/lighting-app/lighting-common/lighting-app.matter

```diff
@@ -1012,6 +1012,21 @@ client cluster OnOff = 6 {
   attribute enum8 startUpOnOff = 16387;
   readonly global attribute bitmap32 featureMap = 65532;
   readonly global attribute int16u clusterRevision = 65533;
+
+  request struct OffWithEffectRequest {
```

Comment:
Should probably not have changes in here.

Reply:
yep, accidental zap generated...